awscli/topics/config-vars.rst

```diff
@@ -245,6 +255,13 @@ Example configuration::
   role_arn=arn:aws:iam:...
   source_profile=development
 
+Example configuration using ``credential_source``::
+
+  # In ~/.aws/config
+  [profile crossaccount]
+  role_arn=arn:aws:iam:...
+  credential_source=Ec2InstanceMetadata
+
```

Comment:
I wonder if it makes sense to explain what this example does exactly in obtaining credentials?